dist/mentio.js

```diff
@@ -328,6 +328,102 @@ angular.module('mentio', [])
                     }
                 }
 
+                function evaluateModelValue(newValue) {
+                    /*jshint maxcomplexity:14 */
+                    /*jshint maxstatements:39 */
+                    // yes this function needs refactoring
+                    if ((!newValue && !(newValue === '' && scope.triggerCharSet.indexOf('') >= 0)) &&
```

Comment:
when model is an empty string, addMenu hasn't yet been called, and triggerCharSet is null. I believe you'd want a null check on scope.triggerCharSet (Setting the model to empty string allows to pop the menu immediately upon focus)